src/legacy/core_plugins/kbn_doc_views/public/views/table.html

```diff
@@ -91,12 +94,8 @@
         <i ng-if="!mapping[field] && field[0] !== '_' && !showArrayInObjectsWarning(doc, field)"
           tooltip-placement="top"
           tooltip="{{ ::'kbnDocViews.table.noCachedMappingForThisFieldTooltip' | i18n: { defaultMessage: 'No cached mapping for this field. Refresh field list from the Management > Index Patterns page' } }}"
-          class="fa fa-warning text-color-warning kbnDocViewer__noMapping"></i>
-        <i ng-if="showArrayInObjectsWarning(doc, field)"
-          tooltip-placement="top"
-          tooltip="{{ ::'kbnDocViews.table.objectsInArraysAreNotWellSupportedTooltip' | i18n: { defaultMessage: 'Objects in arrays are not well supported.' } }}"
-          class="fa fa-warning text-color-warning kbnDocViewer__objectArray"></i>
-        <div class="kbnDocViewer__value" ng-bind-html="typeof(formatted[field]) === 'undefined' ? hit[field] : formatted[field] | trustAsHtml"></div>
```

Comment:
I've remove the 'kbnDocViews.table.objectsInArraysAreNotWellSupportedTooltip' warning, since it won't be true soon, and to make space for the collapse icon